docs/team/irfan.md

```diff
@@ -3,44 +3,74 @@ layout: page
 title: Irfan's Project Portfolio Page
 ---
 
-### Project: AddressBook Level 3
+### Project: CallMeMaybe
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+CallMeMaybe (CMM) is a **desktop app** centered for Telemarketers in aiding them in customer contact management.
+The in-built tracking functionality serves as a reminder to reach back on previously unreachable customers.
+Importing and exporting of existing customer database is also supported by CMM to facilitate team-based environments.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. This It is written in Java, and has about
```

Comment:
Minor issue. There seems to be a typo in the sentence starting with 'This It....'.